packages/mobile/src/screens/edit-track-screen/types.ts

```diff
@@ -12,11 +12,15 @@ export type FormValues = Omit<TrackMetadataForUpload, 'bpm'> & {
   }
   trackArtwork?: string
   bpm?: string
+  isUpload?: boolean
 }
 
 export type EditTrackScreenProps = {
   onSubmit: (values: TrackMetadataForUpload) => void
-  initialValues: TrackMetadataForUpload & { trackArtwork?: string }
+  initialValues: TrackMetadataForUpload & {
+    trackArtwork?: string
+    isUpload?: boolean
```

Comment:
maybe the name `TrackMetadataForUpload` is getting a little overloaded

Reply:
would honestly love a rethink with this type values and where they are located